### ISSUE TITLE
Add ThinkPad X250 Trackpoint to hwdb

### ISSUE THREAD

**Issue by stsydow** [systemd/systemd#5193]
Add ThinkPad X250 trackpoint to hwdb.
Without the trackpoint is very insensitive.

To fix is one just needs to add the following:

```
--- a/hwdb/70-pointingstick.hwdb
+++ b/hwdb.d/70-pointingstick.hwdb
@@ -97,6 +97,8 @@
 evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX230Tablet:*
 # Lenovo Thinkpad X240
 evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX240:*
+# Lenovo Thinkpad X250
+evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX250:*
 # Lenovo Thinkpad T440s
 evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadT440s:*
 # Lenovo Thinkpad T540p
```

(Running systemd v231 on Fedora v25)

**Comment by keszybz** [comment]
Sure. Can you prepare a pull request? This kind of change you might even do through the github web interface.

**Comment by stsydow** [comment]
Has already been fixed in e5bc3f4fdc2ac2f68b781adc958e9570e577538b.
Sorry!
